setup.py

```diff
@@ -3,14 +3,24 @@
 with open("README.md", "r") as fh:
     long_description = fh.read()
 
+REQUIRES = [
+  "django>=2.2",
+]
+
+VERSION = "0.0.2"
+URL = "https://veryeasyai.com/"
+
 setuptools.setup(
     name="very_easy_recommendation_engine",
-    version="0.0.1",
+    version=VERSION,
     author="veryeasyai",
+    url=URL,
     description="A Very Easy Recommendation Engine for Django",
     long_description=long_description,
     long_description_content_type="text/markdown",
     packages=setuptools.find_packages(),
+    keywords="django, AI, veryeasyai",
+    license="MIT",
```

Comment:
Looks good @autn !

Want to add an MIT license for us? Something like this: https://github.com/plaid/plaid-python/blob/master/LICENSE